stories/module-analytics.stories.js

```diff
@@ -124,6 +128,51 @@ storiesOf( 'Analytics Module', module )
 			</WithTestRegistry>
 		);
 	} )
+	.add( 'Property Select including GA4 properties', () => {
+		// Have to import like with tests
+		enabledFeatures.add( 'ga4setup' );
+
+		const { accounts, properties, profiles } = fixtures.accountsPropertiesProfiles;
+		/* eslint-disable sitekit/acronym-case */
+		const accountID = properties[ 0 ].accountId;
+		const propertyID = profiles[ 0 ].webPropertyId;
+		/* eslint-enable */
+		const setupRegistry = ( { dispatch } ) => {
+			dispatch( STORE_NAME ).receiveGetAccounts( accounts );
+			dispatch( STORE_NAME ).finishResolution( 'getAccounts', [] );
+
+			// eslint-disable-next-line sitekit/acronym-case
+			dispatch( STORE_NAME ).receiveGetProperties( properties, { accountID: properties[ 0 ].accountId } );
+			// I believe we don't need profiles for this story, do we? Let's get rid of it if we don't need it
+			dispatch( STORE_NAME ).receiveGetProfiles( profiles, {
+				accountID,
+				propertyID,
+			} );
+
+			dispatch( STORE_NAME ).receiveGetSettings( {
+				accountID,
+			} );
+			dispatch( MODULES_ANALYTICS_4 ).receiveGetProperties(
+				propertiesGA4,
+				{ accountID }
+			);
+		};
+
+		return (
+			<WithTestRegistry
+				callback={ setupRegistry }
+				// This does not work
+				features={ [ 'ga4setup' ] }
```

Comment:
This does not cause the feature flag to be set in `analytics/datastore/properties.js`

Reply:
It uses a slightly different approach... we may need to revisit it i think :thinking: but it's out of scope for this ticket.